backend/src/metrics.rs

```diff
@@ -43,6 +43,11 @@ const HTTP_REQUESTS : MetricDesc = MetricDesc {
     help: "Number of incoming HTTP requests",
     unit: None,
 };
+const BUILD_INFO : MetricDesc = MetricDesc {
+    name: "build_info",
+    help: "Different information about the app",
```

Comment:
Different from what? :thinking: I'm not sure this is the right word here, but I'm also not sure what exactly you want to say. "Various" maybe? Or "miscellaneous"?

But more saliently, why does this not just say `"Information about hte app's build"`?